src/applications/vaos/newAppointmentFlow.js

```diff
@@ -114,18 +114,18 @@ export default {
               }
             }
           }
-
-          // If no CC enabled systems and toc is podiatry, show modal
-          if (isPodiatry(state)) {
-            dispatch(showTypeOfCareUnavailableModal());
-            return 'typeOfCare';
-          }
         } catch (e) {
           captureError(e);
           Sentry.captureMessage(
             'Community Care eligibility check failed with errors',
           );
         }
+
+        // If no CC enabled systems and toc is podiatry, show modal
+        if (isPodiatry(state)) {
```

Comment:
This fixes a bug where if there's a CC error, we continue on to the VA facility page. That's right in all cases except Podiatry, which doesn't support VA facilities yet.